modules/ui/fields/input.js

```diff
@@ -18,12 +18,14 @@ export {
     uiFieldText as uiFieldEmail,
     uiFieldText as uiFieldIdentifier,
     uiFieldText as uiFieldNumber,
+    uiFieldText as uiFieldSchedule,
```

Comment:
Actually it would be nice to keep the dropdown button for some common values like `24/7` that YoHours doesn’t support. But the combo field doesn’t support an external link button as far as I know.